tutorials/audio_tasks/speech_enhancement/Speech_Enhancement_with_NeMo.ipynb

```diff
@@ -0,0 +1,1162 @@
+    "\n",
+    "*Disclamer:*\n",
+    "User is responsible for checking the content of datasets and the applicable licenses and determining if suitable for the intended use."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50525ece",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "5. Restart the runtime (Runtime -> Restart Runtime) for any upgraded packages to take effect\n",
+    "\"\"\"\n",
+    "\n",
+    "GIT_USER = 'anteju' # TODO: change to 'NVIDIA'\n",
+    "GIT_BRANCH = 'dev/se-tutorial' # TODO: change to 'main'\n",
+    "\n",
+    "if 'google.colab' in str(get_ipython()):\n",
+    "\n",
+    "    # Install dependencies\n",
+    "    !pip install wget\n",
+    "    !apt-get install sox libsndfile1 ffmpeg\n",
+    "    !pip install text-unidecode\n",
+    "    !pip install matplotlib>=3.3.2\n",
+    "\n",
+    "    ## Install NeMo\n",
+    "    !python -m pip install git+https://github.com/{GIT_USER}/NeMo.git@{GIT_BRANCH}#egg=nemo_toolkit[all]\n",
+    "\n",
+    "    ## Install TorchAudio\n",
+    "    !pip install torchaudio>=0.13.0 -f https://download.pytorch.org/whl/torch_stable.html"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "765e8b8a",
+   "metadata": {},
+   "source": [
+    "The following cell will take care of the necessary imports and prepare utility functions used throughout the notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9248dd5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import librosa\n",
+    "import os\n",
+    "import torch\n",
+    "import tqdm\n",
+    "\n",
+    "import IPython.display as ipd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pytorch_lightning as pl\n",
+    "import soundfile as sf\n",
+    "\n",
+    "from pathlib import Path\n",
+    "from torchmetrics.functional.audio import signal_distortion_ratio, scale_invariant_signal_distortion_ratio\n",
+    "\n",
+    "from nemo.utils.notebook_utils import download_an4\n",
+    "from nemo.collections.asr.parts.preprocessing.segment import AudioSegment\n",
+    "from nemo.collections.asr.parts.utils.manifest_utils import read_manifest, write_manifest\n",
+    "\n",
+    "\n",
+    "# Used to download data processing scripts\n",
+    "USER = 'anteju' # TODO: change to 'NVIDIA'\n",
```

Comment:
Don't have two locations for user and branch. Also best to follow the normal format and drop user.